2199/3/app/onliner_page_parser.rb

```diff
@@ -0,0 +1,53 @@
+require 'pry'
+require_relative '../app/comment_parser'
+require_relative 'capybara_initializer'
+
+# class for parsing
+class OnlinerPageParser
+  COMMENTS_TO_TAKE = 50
+
+  def initialize(link)
+    @link = link
+  end
+
+  def browser
+    @browser ||= Capybara.current_session
+  end
+
+  # :reek:TooManyStatements
+  def visit_page
+    attempts = 0
+    begin
+      browser.visit(@link)
+    rescue StandardError => exception
+      attempts += 1
+      sleep(2 * attempts)
```

Comment:
Why did you choose `2*attempts` formula?

Reply:
I use exponential backoff. I did something wrong??